components/FAQ/FAQ.tsx

```diff
@@ -0,0 +1,38 @@
+'use client'
+
+import { Montserrat } from "next/font/google";
+import LinkButton from "@/LinkButton"
+import React, { useState } from 'react'
+import FAQAccordion from "./FAQAccordion"
+import data from "./FAQ.json"
+
+const montserrat = Montserrat({ subsets: ['cyrillic'], weight: ['400', '500', '700'] });
```

Comment:
The font should be loaded in the layout. 

https://nextjs.org/docs/app/building-your-application/optimizing/fonts#with-tailwind-css